fractal_server/app/routes/api/v1/_aux_functions.py

```diff
@@ -378,3 +380,56 @@ def _get_active_jobs_statement() -> SelectOfScalar:
         )
     )
     return stm
+
+
+async def _workflow_insert_task(
+    workflow_id: int,
+    task_id: int,
```

Comment:
Why having these as positional arguments, if in practice we always call them as kwargs? Shall we move the `*,` up by two lines?